db/migrate/20230112214407_create_membership_requests.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateMembershipRequests < ActiveRecord::Migration[5.2]
+  def change
+    create_table :membership_requests do |t|
+      t.references :user
+      t.references :organization
+      t.string :status, null: false
+      t.string :closed_by_user_id, null: true
+      t.datetime :closed_at_datetime
```

Comment:
```suggestion
      t.datetime :closed_at
```
I don't think the _datetime is necessary